atom/app/command_line_args.cc

```diff
@@ -42,23 +42,35 @@ const char* blacklist[] = {
   "inspect",
   "inspect-brk",
 
+
   // chromium switches:
-  // find ./ -name "*switches.cc" \
-  //   | xargs grep -P --no-filename "\"\S+\";" \
-  //   | perl -pe 's|^.*?"(\S+)";|  "$1",|' \
-  //   | sort | uniq
-  "?",
+  // find ./ \( -name "*switches.cc" -o -name "*switch_list.h" \) \
+  //   | xargs grep -P --no-filename '"\S+"[);]$' \
+  //   | perl -pe 's|^.*?"(\S+)"[);]$|  "$1",|'; \
+  //   | LANG="C" sort | uniq
+  "/prefetch:1",
+  "/prefetch:2",
+  "/prefetch:3",
+  "/prefetch:4",
+  "/prefetch:5",
+  "/prefetch:6",
+  "/prefetch:8",
   "0",
+  "?",
```

Comment:
This looks weird

Reply:
Answered [here](https://github.com/electron/electron/pull/12807#discussion_r188110701)